src/common/libkvs/kvs.c

```diff
@@ -27,6 +27,29 @@
 #endif
 #include <flux/core.h>
 
+int flux_kvs_namespace_create (flux_t *h, const char *namespace, int flags)
```

Comment:
This should probably return a `flux_future_t *` so the caller can decide if they want to handle the result synchronously or asynchronously in a continuation.